src/components/Footer/Footer.tsx

```diff
@@ -36,6 +40,47 @@ function Footer({ socialLinks }: Props) {
               ))}
           </ul>
         </Col>
+        <Col size={6}>
+          <ul className="p-list">
```

Comment:
These should be passed as props as the links might not always be the same between sites

Reply:
I made a change :)